.gitignore

```diff
@@ -1,11 +1,14 @@
+# general
 .vscode
 .idea
 *.code-workspace
 .DS_Store
+._.DS_Store
 
 # helps Lennart configure his vim
 pyrightconfig.json
 
-.DS_Store
-._.DS_Store
 /private/
+
+# TODO: temp
+app/web-frontend/proto/
```

Comment:
I listen your opinions about this

Reply:
I don't think there should be anything in the folder `app/web-frontend`?

Reply:
yap, tons of files 😅 (for real)

Reply:
![image](https://github.com/Couchers-org/couchers/assets/16777161/b740825e-b920-4307-afd9-d179511e3950)

Reply:
You can delete the whole folder.

While the repos were separate, the suggested way to work on both was to clone web-frontend into that folder. But now that’s not a thing anymore, so you can delete that whole folder.